app/sass/_custom.scss

```diff
@@ -303,3 +303,10 @@ footer li {
   }
 }
 
+.meetup.primary {
+  background-color: #fff;
+}
+
+.meetup.secondary {
```

Comment:
@criana I've added the new meetup feeds and added support for each meetup feed to have custom css classes.  I've set it up so we've got `primary meetup` class for our's as well as `secondary meetup` for everything else. I added a terrible style as a placeholder, can you pull this branch `more-meetups` make it pretty and then push it back up with your changes? After that we can merge this into `master` for deploy to production.